libazureinit/src/provision/ssh.rs

```diff
@@ -37,17 +39,80 @@ pub(crate) fn provision_ssh(
     Ok(())
 }
 
+pub(crate) fn get_authorized_keys_path_from_sshd() -> io::Result<Option<PathBuf>>
```

Comment:
`cargo check` gives warning below. Where is the function being called?
```
warning: function `get_authorized_keys_path_from_sshd` is never used
  --> libazureinit/src/provision/ssh.rs:42:15
   |
42 | pub(crate) fn get_authorized_keys_path_from_sshd() -> io::Result<Option<PathBuf>>
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default

warning: `libazureinit` (lib) generated 1 warning
```

Reply:
This function will be used in provision_ssh in ssh.rs to provision the ssh keys on line 30. I will make the changes to use get_authorized_keys_path_from_sshd.